winfetch.ps1

```diff
@@ -366,46 +365,79 @@ $img = if (-not $noimage) {
+                "${e}[${t};34mlllllllllllllll   lllllllllllllll"
+            )
+        } elseif ($logo -eq "Windows 10" -Or $logo -eq "Windows 8.1" -Or $logo -eq "Windows 8") {
+            @(
+                "${e}[${t};34m                    ....,,:;+ccllll"
+                "${e}[${t};34m      ...,,+:;  cllllllllllllllllll"
+                "${e}[${t};34m,cclllllllllll  lllllllllllllllllll"
+                "${e}[${t};34mllllllllllllll  lllllllllllllllllll"
+                "${e}[${t};34mllllllllllllll  lllllllllllllllllll"
+                "${e}[${t};34mllllllllllllll  lllllllllllllllllll"
+                "${e}[${t};34mllllllllllllll  lllllllllllllllllll"
+                "${e}[${t};34mllllllllllllll  lllllllllllllllllll"
+                "${e}[${t};34m                                   "
+                "${e}[${t};34mllllllllllllll  lllllllllllllllllll"
+                "${e}[${t};34mllllllllllllll  lllllllllllllllllll"
+                "${e}[${t};34mllllllllllllll  lllllllllllllllllll"
+                "${e}[${t};34mllllllllllllll  lllllllllllllllllll"
+                "${e}[${t};34mllllllllllllll  lllllllllllllllllll"
+                "${e}[${t};34m``'ccllllllllll  lllllllllllllllllll"
+                "${e}[${t};34m      ``' \\*::  :ccllllllllllllllll"
+                "${e}[${t};34m                       ````````''*::cll"
+                "${e}[${t};34m                                 ````"
+            )
+        } elseif ($logo -eq "Windows 7" -Or $logo -eq "Windows Vista" -Or $logo -eq "Windows XP") {
+            @(
+                "${e}[${t};31m        ,.=:!!t3Z3z.,               "
+                "${e}[${t};31m       :tt:::tt333EE3               "
+                "${e}[${t};31m       Et:::ztt33EEE  ${e}[32m@Ee.,      ..,"
+                "${e}[${t};31m      ;tt:::tt333EE7 ${e}[32m;EEEEEEttttt33#"
+                "${e}[${t};31m     :Et:::zt333EEQ. ${e}[32mSEEEEEttttt33QL"
+                "${e}[${t};31m     it::::tt333EEF ${e}[32m@EEEEEEttttt33F "
+                "${e}[${t};31m    ;3=*^``````'*4EEV ${e}[32m:EEEEEEttttt33@. "
+                "${e}[${t};34m    ,.=::::it=., ${e}[31m`` ${e}[32m@EEEEEEtttz33QF  "
+                "${e}[${t};34m   ;::::::::zt33)   ${e}[32m'4EEEtttji3P*   "
+                "${e}[${t};34m  :t::::::::tt33 ${e}[33m:Z3z..  ${e}[32m```` ${e}[33m,..g.   "
+                "${e}[${t};34m  i::::::::zt33F ${e}[33mAEEEtttt::::ztF    "
+                "${e}[${t};34m ;:::::::::t33V ${e}[33m;EEEttttt::::t3     "
+                "${e}[${t};34m E::::::::zt33L ${e}[33m@EEEtttt::::z3F     "
+                "${e}[${t};34m{3=*^``````'*4E3) ${e}[33m;EEEtttt:::::tZ``     "
+                "${e}[${t};34m            `` ${e}[33m:EEEEtttt::::z7       "
+                "${e}[${t};33m                'VEzjt:;;z>*``       "
+            )
+        } else {
+            Write-Error 'The only version logos supported are Windows 11, Windows 10/8.1/8, and Windows 7/Vista/XP.'
+            exit 1
```

Comment:
I'm a little unsure if this is the right response, we might be better off just defaulting to the legacy logo if the platform is not Windows 11/10/8? It'd be simpler and safer for unknown versions or edge cases. If you really want to be explicit, you could disable the image and keep going, maybe with a warning as well?

Reply:
@jcwillox, as per Line 374 we do actually default to the legacy logo. This error only appears if you explcitly use the new logo parameter and provide an invalid version of windows (eg. `winfetch.ps1 -o "Windows 95"`)

Reply:
Oh great, totally missed that